sdk/batch/batch/test/batchClient.spec.ts

```diff
@@ -0,0 +1,1232 @@
+        "HelloWorldNodeSDKTestTask2",
+        "stderr.txt"
+      );
+
+      assert.equal(result._response.status, 200);
+    });
+
+    it("should get file from task successfully", (done) => {
+      client.file
+        .getFromTask("HelloWorldJobNodeSDKTest", "HelloWorldNodeSDKTestTask2", "stdout.txt")
+        .then((result) => {
+          assert.equal(result._response.status, 200);
+          readStreamToBuffer(result.readableStreamBody!, function (_err, buff) {
+            assert.isAtLeast(buff.length, 1);
+            done();
+          });
+        })
+        .catch((error) => {
+          assert.fail(error);
+        });
+    });
+
+    it("should delete file from task successfully", async () => {
+      const result = await client.file.deleteFromTask(
+        "HelloWorldJobNodeSDKTest",
+        "HelloWorldNodeSDKTestTask2",
+        "stderr.txt"
+      );
+
+      assert.equal(result._response.status, 200);
+    });
+
+    it("should re-list compute nodes successfully", async () => {
+      const result = await client.computeNode.list("nodesdktestpool1");
+
+      assert.isAtLeast(result.length, 1);
+      compute_nodes = result.map(function (x) {
+        return x.id!;
+      });
+      //wait(100000);
+      // if (!suite.isPlayback) {
+      //   console.log('Waiting for nodes to be ready...')
+      //   setTimeout(function () {
+      //     done();
+      //   }, 100000);
+      // } else {
+      //   done();
+      // }
+    });
+
+    it("should list files from compute node successfully", async () => {
+      const result = await client.file.listFromComputeNode("nodesdktestpool1", compute_nodes[1]);
+
+      assert.isAtLeast(result.length, 1);
+      assert.equal(result._response.status, 200);
+    });
+
+    it("should get file properties from node successfully", async () => {
+      const result = await client.file.getPropertiesFromComputeNode(
+        "nodesdktestpool1",
+        compute_nodes[1],
+        "startup/wd/hello.txt"
+      );
+
+      assert.equal(result._response.status, 200);
+    });
+
+    it("should get file from node successfully", (done) => {
+      client.file
+        .getFromComputeNode("nodesdktestpool1", compute_nodes[1], "startup/wd/hello.txt")
+        .then((result) => {
+          assert.equal(result._response.status, 200);
+          readStreamToBuffer(result.readableStreamBody!, function (_err, buff) {
+            assert.isAtLeast(buff.length, 1);
+            done();
+          });
+        })
+        .catch((error) => {
+          assert.fail(error);
+        });
+    });
+
+    it("should delete file from node successfully", async () => {
+      const result = await client.file.deleteFromComputeNode(
+        "nodesdktestpool1",
+        compute_nodes[1],
+        "startup/wd/hello.txt"
+      );
+
+      assert.equal(result._response.status, 200);
+    });
+
+    // the application is not added by the tests and should be added by the tester manually
```

Comment:
Is this a prerequisite for running the test? If so It may be helpful to make it more explicit by adding something like `// Pre-requisite: `